index.js

```diff
@@ -11,16 +12,20 @@ module.exports = {
     var resolverConfig = config['ember-resolver'] || {};
 
     return Object.assign({
-      /* Add default feature flags here */
-      EMBER_RESOLVER_MODULE_UNIFICATION: false
+      /* Add default feature flags here, for now there is none */
     }, resolverConfig.features);
   },
 
   init: function() {
     this._super.init.apply(this, arguments);
     this.options = this.options || {};
-
+    if (process.env.EMBER_RESOLVER_MODULE_UNIFICATION) {
+      this.project.isModuleUnification = function () {
```

Comment:
This seems fairly odd to me. Why do we have to monkey patch this?

Reply:
For testing only. Any suggestion?